RPi3.md

```diff
@@ -0,0 +1,278 @@
+
+In this case, we will be using Etcher to flash your microSD card with the downloaded Raspbian Lite OS. Plug in the microSD card, and run Etcher. Select the unzipped Raspbian OS, select your microSD card, and confirm to flash it.
+
+**Step 5** - On this same computer, **⚠️ before you plug the SD card into your Pi**, create an empty file named `ssh` in the boot partition of the SD card.
+
+- On Mac and Linux, use `touch ssh` in the card's root directory via `Terminal`
+- On Windows, use `type nul > ssh` in the card's root directory via `cmd`
+
+**Step 6** - Insert your microSD card and flash drive into the Pi; connect the network cable and power supply.
+
+**Step 7** - From another computer, use an SSH client (`ssh` on Mac and Linux, [PuTTY](https://putty.org) on Windows) to connect to your Raspberry Pi:
+
+- hostname: `raspberrypi.local`
+- username: `pi`
+- password: `raspberry`
+
+So: `ssh pi@raspberrypi.local`.
+
+If `raspberrypi.local` doesn't work, you will have to either look up the Pi's IP address on your router, or run `ifconfig` on the Pi directly for the `eth0` `inet` address.
+
+**Step 8 - ⚠️ IMPORTANT!** - Change your password:
+```bash
+passwd
+```
+
+**Step 9** - Give your Pi a static IP address and a DHCP reservation on your local network, via your router. Optionally, setup WiFi. There are a few different ways to do this and you will find a ton of articles online. Here's a pretty simple one to follow: [Setting up Raspberry Pi WiFi with Static IP on Raspbian Stretch Lite](https://electrondust.com/2017/11/25/setting-raspberry-pi-wifi-static-ip-raspbian-stretch-lite/).
+
+To get your router's IP:
+
+- On Linux: `ip route | grep default`
+- On Mac: `netstat -nr | grep default`
+- On Windows: `ipconfig | findstr /i "Gateway"`
+
+**Step 10** - Log into your router and forward ports `80`, `443`, and `9735` to your Pi's local IP address. Every router is different and you should be able to find instructions for your router by searching the web for "Port Forwarding + {your router make and model}".
+
+**Step 11** - Install `fail2ban` and `git`.
+
+`fail2ban` bans IPs that attempt to connect to your server and show malicious signs. `git` allows you to clone and manage repositories on github.com.
+
+So, open a new terminal window and type the following command:
+```bash
+sudo apt update && sudo apt install -y fail2ban git
+```
+
+**⚠️ Note for beginners:** Run all commands in these instructions **one line at a time**!
+
+**Step 12** - Install `ufw` (Uncomplicated Firewall) and allow only specific ports. UFW is a user-friendly frontend for managing iptables firewall rules and its main goal is to make managing iptables easier, or as the name says: uncomplicated.
+
+Install UFW:
+```bash
+sudo apt install ufw
+```
+
+This command allows SSH connections from your LAN only. Replace `192.168.1.0` with your own subnet:
```

Comment:
We should highlight that last sentence – I remember messing it up by not paying enough attention 🙈